core/lib/protobuf_config/src/proto/config/prover.proto

```diff
@@ -108,4 +108,5 @@ message ProofDataHandler {
   optional uint32 http_port = 1; // required; u16
   optional uint32 proof_generation_timeout_in_secs = 2; // required; s
   optional bool tee_support = 3; // required
```

Comment:
should this be `// optional` then also?

Reply:
Thanks for noticing it. Fixed [here](https://github.com/matter-labs/zksync-era/pull/3112/commits/d943314829de7e41bbbea6bc0d8cf30fdd83a6f1). If you have a strong opinion to make it required, please lemme know @slowli / @haraldh.

I guess we want to have it optional, defaulting to `false` to make it consistent with [this](https://github.com/matter-labs/gitops-kubernetes/blob/04bd23eb714622298c04adb750804ed35a66976e/apps/common/zksync-v2-config/zksync-v2-config.yaml#L616).

If we decide to make it required, we [would need](https://github.com/matter-labs/zksync-era/pull/3112#discussion_r1810406067) to add the new param for all envs using file-based configs beforehand.